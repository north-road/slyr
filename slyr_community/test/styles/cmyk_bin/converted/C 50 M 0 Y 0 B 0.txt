[{'class': 'QgsSimpleFillSymbolLayer',
  'color': '127,255,255,255',
  'outline_color': '255,0,0,255',
  'outline_width': '8',
  'outline_width_unit': 'Point'}]
