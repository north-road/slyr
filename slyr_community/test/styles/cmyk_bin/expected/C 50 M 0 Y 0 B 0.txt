{'levels': [{'color': {'C': 50,
                       'K': 0,
                       'M': 0,
                       'Y': 0,
                       'dither': False,
                       'is_null': False,
                       'stream_offset': '0xb4',
                       'type': 'CmykColor',
                       'version': 4},
             'enabled': True,
             'fill_style': 'solid',
             'locked': False,
             'outline': {'color': {'B': 0,
                                   'G': 0,
                                   'R': 255,
                                   'dither': True,
                                   'is_null': False,
                                   'stream_offset': '0x71',
                                   'type': 'RgbColor',
                                   'version': 1},
                         'enabled': True,
                         'line_type': 'solid',
                         'locked': False,
                         'stream_offset': '0x5f',
                         'symbol_level': 0,
                         'type': 'SimpleLineSymbol',
                         'version': 1,
                         'width': 8.0},
             'stream_offset': '0x4d',
             'symbol_level': 0,
             'type': 'SimpleFillSymbol',
             'version': 1}],
 'stream_offset': '0x0',
 'symbol_level': 0,
 'type': 'MultiLayerFillSymbol',
 'version': 2}
